File: app/src/main/java/com/project/scratchstudio/kith_andoid/SetInternalData/SetUserIdAndToken.java
package com.project.scratchstudio.kith_andoid.SetInternalData;

import android.content.SharedPreferences;

import com.project.scratchstudio.kith_andoid.Activities.HomeActivity;

public class SetUserIdAndToken implements ISetInternalData {

    @Override
    public int set(SharedPreferences sp) {
        SharedPreferences.Editor ed;
        ed = sp.edit();
        ed.putInt("cur_user_id", HomeActivity.getMainUser().getId());
        ed.putString("cur_user_token", HomeActivity.getMainUser().getToken());
        ed.putString("cur_user_password", HomeActivity.getMainUser().getPassword());
        ed.apply();
        return 0;
    }

}

File: app/src/main/java/com/project/scratchstudio/kith_andoid/SetInternalData/SetUserIdAndToken.kt
package com.project.scratchstudio.kith_andoid.SetInternalData

import android.content.SharedPreferences

import com.project.scratchstudio.kith_andoid.Activities.HomeActivity

class SetUserIdAndToken : ISetInternalData {

    override fun set(sp: SharedPreferences): Int {
        val ed: SharedPreferences.Editor = sp.edit()
        ed.putInt("cur_user_id", HomeActivity.mainUser!!.id)
        ed.putString("user_token", HomeActivity.mainUser!!.token)
        ed.putString("cur_user_password", HomeActivity.mainUser!!.password)
        ed.apply()
        return 0
    }

}
